features/support/run_helpers.rb

```diff
@@ -36,7 +36,7 @@ def integer_output_of command
 
 
 def git_town_command? command
-  %w(extract hack kill pr prune-branches repo ship sync-fork sync town).any? do |subcommand|
+  %w(extract hack kill pull-request prune-branches repo ship sync-fork sync town).any? do |subcommand|
```

Comment:
Aaahh, how could I, the guy who is so obsessed with alphabetic sorting, have missed this! Fixed now